Engine/gfx/ali3dogl.cpp

```diff
@@ -933,7 +947,9 @@ bool OGLGraphicsDriver::GetCopyOfScreenIntoBitmap(Bitmap *destination, bool at_n
   }
   else
   {
-#if !AGS_OPENGL_ES2
+#if AGS_OPENGL_ES2
+    glBindFramebufferEXT(GL_FRAMEBUFFER_EXT, _screenFramebuffer);
+#else
```

Comment:
This was how it was before the changes, so I restored it. I am not 100% sure if it's necessary to do this.